java/src/main/java/actorworkflow/Customer.java

```diff
@@ -0,0 +1,58 @@
+package actorworkflow;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import java.util.ArrayList;
+
+@JsonInclude(Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record Customer(
```

Comment:
retarding use of "record", is sample supposed to be running on jdk14+? our sdk samples run on 11+

Reply:
I do have it set to run on 14+. Do you think I should downgrade to 11+?

Reply:
imho yes but up to you. our samples compile against 8 iirc